web/src/components/core/OptionsPicker.jsx

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) [2024] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+
+/**
+ * Wrapper for OptionsPicker options
+ * @component
+ *
+ * @param {object} props
+ * @param {string} [props.title] - Text to be used as option title
+ * @param {string} [props.body] - Text to be used as option body
```

Comment:
why title and body is in square brackets? it looks like mandatory params to me and not optional.

Reply:
No, they are not. At least one of them should be present, of course. 

So, maybe we can go for making title mandatory and body optional.

Thanks for catching it. Good to see you reviewing these pieces of code too.

Reply:
well, it is partly learning new stuff as I am not much familiar with jsdoc, so I need to read first how it works.